distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/AbstractIntegrationTest.java

```diff
@@ -416,7 +416,8 @@ protected UrlResourceReaderConfigurator getUrlResourceReaderConfigurator() {
                 mavenBundle("ddf.test.thirdparty", "rest-assured").versionAsInProject(),
                 wrappedBundle(mavenBundle("com.google.guava",
                         "guava").versionAsInProject()).exports("*;version=18.0"),
-                wrappedBundle(mavenBundle("ddf.catalog.core", "versioning-common").versionAsInProject()));
+                wrappedBundle(mavenBundle("ddf.catalog.core", "versioning-common").versionAsInProject()),
+                wrappedBundle(mavenBundle("ddf.catalog.core", "catalog-core-api-impl").versionAsInProject()));
```

Comment:
no - you cannot add `catalog-core-api-impl` as a wrappedBundle as it can cause weird deployment issues - same for the `versioning-common` bundle.  Your tests shouldn't require access to these directly - if they do we need to update them.